smithy-aws-cloudformation-traits/src/main/resources/META-INF/smithy/aws.cloudformation.smithy

```diff
@@ -34,6 +34,15 @@ string cfnName
 )
 structure cfnExcludeProperty {}
 
+/// Indicates that a structure member has a default value
+/// for the property of the CloudFormation resource.
+@unstable
+@trait(
+    selector: "structure > member",
+    breakingChanges: [{change: "presence"}]
```

Comment:
In addition to the other comment, this selector should possibly also be locked down with an exclusion of `@notProperty`. That is, if the same member is explicitly marked as not a property, then there's no meaning for `@cfnDefaultValue` to be there.

Traits don't need to be fully locked down by selector, but I think this is low hanging fruit.